examples/trials/mnist-pytorch/config_detailed.yml

```diff
@@ -0,0 +1,42 @@
+# This example shows more configurable fields comparing to the minimal "config.yml"
+# You can use "nnictl create --config config_detailed.yml" to launch this experiment.
+# If you see an error message saying "port 8080 is used", use "nnictl stop --all" to stop previous experiment.
+
+# Hyper-parameter search space can either be configured here or in a seperate file.
+# "config.yml" shows how to specify a seperate search space file.
+# The common schema of search space is documented here:
+#   https://nni.readthedocs.io/en/stable/Tutorial/SearchSpaceSpec.html
+searchSpace:
+  batch_size:
+    _type: choice
+    _value: [16, 32, 64, 128]
+  hidden_size:
+    _type: choice
+    _value: [128, 256, 512, 1024]
+  lr:
+    _type: choice
+    _value: [0.0001, 0.001, 0.01, 0.1]
+  momentum:
+    _type: uniform
+    _value: [0, 1]
+
```

Comment:
yeah, "experimentName" should be put here, we can tell users that they can omit it if they don't want to write it.